pkg/state/accounts_data_storage.go

```diff
@@ -282,6 +282,79 @@ func (s *accountsDataStorage) retrieveEntries(addr proto.Address) ([]proto.DataE
 	return entries, nil
 }
 
+func (s *accountsDataStorage) retrieveEntriesAtHeight(addr proto.Address, height uint64) ([]proto.DataEntry, error) {
```

Comment:
This is actually unused for now, because the caller `stateManager.RetrieveEntriesAtHeight` is also unused.